hub/powertoys/newplus.md

```diff
@@ -0,0 +1,43 @@
+---
+title: PowerToys New+ for Windows
+description: A tool that enables you to create files and folders from an easily personalized set of templates, directly from the File Explorer context menu.
+ms.date: 07/13/2024
+ms.topic: article
+no-loc: [PowerToys, Windows, New+, New, NewPlus, Win]
+---
+
+# New+
+
+**PowerToys New+** enables you to create files and folders from an easily personalized set of templates, directly from the File Explorer context menu.
+
+## Getting started
+
+### How to enable New+
+
+To start using New+, enable New+ in the PowerToys settings.
+
+### How to create a new object using New+
+
+To create a new item within a folder, right-click on the background of the folder to bring up the context menu. From there, click on the “New+” option and then select the template you were looking for.
```

Comment:
In other PowerToys docs, we just say "right-click on the folder" or "right-click on the file". See the Registry Preview docs (https://learn.microsoft.com/en-us/windows/powertoys/registry-preview) for an example.
```suggestion
To create a new item within a folder, right-click on the folder to bring up the context menu. From there, click on the “New+” option and then select the template you were looking for.
```